_posts/2022-08-05-hacking-for-cruise-control-ui.md

```diff
@@ -0,0 +1,241 @@
+|          | +------>  * Operator *
+|          |            *        *              *  *         *  *          *  *
++----------+               *  *              *        *   *        *    *        *
+                                            * Zookeeper* * Zookeeper*  * Zookeeper*
+                                            *   Pod    * *   Pod    *  *   Pod    *
+                                             *        *   *        *    *        *
+                                                *  *         *  *          *  *
+
+Kubernetes Land                                          Kafka Cluster
+```
+Since the [Cruise Control UI](https://github.com/linkedin/cruise-control-ui) does not support HTTPs and Strimzi does not support creating custom Cruise Control API user roles we will need to disable Cruise Control SSL and HTTP basic authentication settings. 
+
+We can do so by configuring a Kafka custom resource like this:
+
+``` 
+# kafka-cruise-control.yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  cruiseControl:
+    config:
+      webserver.security.enable: false
+      webserver.ssl.enable: false  
+```
+
+## Cruise Control UI
+
+With our Strimzi managed Kafka cluster deployed, we can now create a Cruise Control UI instance to run alongside it.
+
+```
+    *  *                   *  *
+ *       *              *        *
+*  Cruise  *   +---->  *  Cruise  *
+*  Control *   <----+  *  Control *
+ *   UI  *              *        *
+    *  *                   *  *
+                            ^                   *  *         *  *          *  *
+                            |                *        *   *        *    *        *
++----------+                |               *  Kafka   * *  Kafka   *  *  Kafka   *
+|          |               *  *             *   Pod    * *   Pod    *  *   Pod    *
+|  Kafka   |            *        *           *        *   *        *    *        *
+| Resource | <------+  * Cluster  * ----->      *  *         *  *          *  *
+|          | +------>  * Operator *
+|          |            *        *              *  *         *  *          *  *
++----------+               *  *              *        *   *        *    *        *
+                                            * Zookeeper* * Zookeeper*  * Zookeeper*
+                                            *   Pod    * *   Pod    *  *   Pod    *
+                                             *        *   *        *    *        *
+                                                *  *         *  *          *  *
+
+Kubernetes Land                                          Kafka Cluster
+```
+
+### Inside the Cruise Control UI pod
+
+Following the instructions on the [Cruise Control UI wiki] (https://github.com/linkedin/cruise-control-ui/wiki/CCFE-(Dev-Mode)---Docker), we can configure an Nginx server as a reverse-proxy to forward Cruise Control UI requests to the Cruise Control REST API.
```

Comment:
The Markdown link seems ot be wrong -> but the URL itself doesn't work for me either.

```suggestion
Following the instructions on the [Cruise Control UI wiki](https://github.com/linkedin/cruise-control-ui/wiki/CCFE-(Dev-Mode)---Docker), we can configure an Nginx server as a reverse-proxy to forward Cruise Control UI requests to the Cruise Control REST API.
```